### PR TITLE
doc: update doc for HTTP file.data to server

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/4144

Describe changes:
- doc: update doc for HTTP file.data to server

### CODE REVIEW

File: doc/userguide/rules/http-keywords.rst [hunk 1]
```diff
@@ -66,7 +66,7 @@ http.cookie                    http_cookie              Both
 http.response_body             http_server_body         Response
 http.server                    N/A                      Response
 http.location                  N/A                      Response
-file.data                      file_data (*)            Response
+file.data                      file_data (*)            Both
```
Comment (corradoleita) [inline doc/userguide/rules/http-keywords.rst:69]: This table contains only fields that are response keywords. I think you should add it also to the table above listing the request keyword.

File: doc/userguide/rules/http-keywords.rst [hunk 2]
```diff
@@ -728,6 +728,8 @@ file.data
 
 With ``file.data``, the HTTP response body is inspected, just like
 with ``http.response_body``. The ``file.data`` keyword is a sticky buffer.
+``file.data`` also works for HTTP request body (and can be used in other
+protocols than HTTP1).
```
Comment (corradoleita) [inline doc/userguide/rules/http-keywords.rst:732]: I think "and can be used in protocols other than HTTP1" sounds better

File: doc/userguide/rules/http-keywords.rst (hunk 1 above)
Comment (catenacyber) [reply doc/userguide/rules/http-keywords.rst:69]: indeed

File: doc/userguide/rules/http-keywords.rst (hunk 2 above)
Comment (catenacyber) [reply doc/userguide/rules/http-keywords.rst:732]: ok

**Pull request closed without merging**